lib/ansible/modules/windows/win_package.ps1

```diff
@@ -850,7 +850,7 @@ function Set-TargetResource
                     if($process)
                     {
                         $exitCode = $process.ExitCode
-                        $result.exit_code = $exitCode
+                        $result["exit_code"] = $exitCode
```

Comment:
I would keep it the same as it is, I prefer `$result.exit_code` myself but it depends on the context. Looks like this module uses that form so I would keep it the way it was.